koi.gemspec

```diff
@@ -18,23 +18,17 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency 'mime-types', '~> 2.3'
+  s.add_dependency 'mime-types'#, '~> 2.3'
 
-  s.add_dependency 'rails', '~> 4.2.1'
+  s.add_dependency 'rails', '~> 5.1.0'
 
-  s.add_dependency 'rails-observers'
-
-  s.add_dependency 'active_model_serializers', '~> 0.8.2'
+  s.add_dependency 'active_model_serializers'#, '~> 0.8.2'
```

Comment:
Should we be restricting the version range for these dependencies? Would need to determine what the version being used is and add that.